src/app/models/admin_feedback.server.model.ts

```diff
@@ -0,0 +1,59 @@
+import { Mongoose, Schema } from 'mongoose'
+
+import { IAdminFeedbackModel, IAdminFeedbackSchema } from '../../types'
+
+import { USER_SCHEMA_ID } from './user.server.model'
+
+// also acts as the model Id
```

Comment:
/nit `// alias of model Id`

Reply:
😅 [ab89573](https://github.com/opengovsg/FormSG/pull/6454/commits/ab895734fb40ccb94be84a3c90659bde87a801b0)